tests/unit-core/test-api.c

```diff
@@ -163,12 +163,16 @@ handler_construct (const jerry_call_info_t *call_info_p, /**< call information *
   TEST_ASSERT (args_cnt == 1);
   TEST_ASSERT (jerry_value_is_boolean (args_p[0]));
   TEST_ASSERT (jerry_get_boolean_value (args_p[0]) == true);
+  TEST_ASSERT(jerry_is_value_false(args_p[0]) == false);
```

Comment:
Space missing before `(`
I would also change `jerry_value_is_boolean && jerry_get_boolean_value` pairs in the tests whenever it is possible.